### DISCUSSION TITLE
IDH node error - SO 2.4.60

### DISCUSSION THREAD

**Discussion by cusohn** [Security-Onion-Solutions/securityonion#12927] [Unsupported Versions] [answered]
Hello,

I encountered an error during the installation of the IHD node on the Security Onion 2.4.60 distributed architecture.

<img width="773" alt="Screenshot 2024-05-03 at 9 30 29 AM" src="https://github.com/Security-Onion-Solutions/securityonion/assets/7014071/ee2b6a31-4df0-4129-8585-d5e63843aedc">

Any solutions or suggestions? Thank you

**Comment by dougburks** [comment]
Are you using our Security Onion ISO image or a network installation? Please note that network installations are not supported:
https://docs.securityonion.net/en/2.4/network-installation.html

If you're using our Security Onion ISO image, are you making any manual changes to the system?

**Reply by cusohn** [reply]
I used the 2.4.60 ISO file to install the system. However, I encountered an error when I tried to restart IDH using 'so-idh-restart'.

**Reply by dougburks** [reply]
I don't see that error on any of my IDH nodes.

Are you making any manual changes to the system?

**Comment by m0duspwnens** [comment, marked as answer]
The IDH node is likely suffering from the same bug as posted in this discussion: https://github.com/Security-Onion-Solutions/securityonion/discussions/12952

Can you try that solution and let us know if it solves the issue?
